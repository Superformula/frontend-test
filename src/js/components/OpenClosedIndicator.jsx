import React, { Component } from 'react';

class OpenClosedIndicator extends Component {

  render() {
    return (
      <div className="open-closed-indicator">
        {closed
            ? <React.Fragment>
                <div className='indicator closed'/>
                closed
            </React.Fragment>
            : <React.Fragment>
                <div className='indicator open'/>
                open now
            </React.Fragment>
        }
      </div>
    );
  }
}

export default OpenClosedIndicator;
