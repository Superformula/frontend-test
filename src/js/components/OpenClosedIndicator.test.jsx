import React from 'react';
import { shallow } from 'enzyme';

import OpenClosedIndicator from './OpenClosedIndicator.jsx';

describe('OpenClosedIndicator', () => {
  it('should render correctly', () => {
    const component = shallow(<OpenClosedIndicator />);

    expect(component).toMatchSnapshot();
  });
});
