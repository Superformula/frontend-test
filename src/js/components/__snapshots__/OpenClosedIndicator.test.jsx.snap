// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OpenClosedIndicator should render correctly 1`] = `
<div
  className="open-closed-indicator"
>
  <div
    className="indicator open"
  />
  open now
</div>
`;
